src/status_im/multiaccounts/recover/core.cljs

```diff
@@ -101,9 +101,15 @@
           constants/path-whisper
           constants/path-default-wallet]
          (fn [result]
-           (let [derived-data (types/json->clj result)]
-             (re-frame/dispatch [::import-multiaccount-success
-                                 root-data derived-data])))))))))
+           (let [derived-data (types/json->clj result)
+                 publicKey (get-in derived-data [constants/path-whisper-keyword :publicKey])]
```

Comment:
we use `kebab-case` rather than `camelCase`

Reply:
Fixed